docs/input4MIPs_files_CMIP6Plus.html

```diff
@@ -21,7 +21,7 @@
 <title>Input4MIPs CMIP6Plus all files</title>
 </head>
 <body>
-<p><h1>Input4MIPs CMIP6Plus files: v6.5.3</h1><p>
+<p><h1>Input4MIPs CMIP6Plus files: v0.1.0</h1><p>
```

Comment:
Ditto with comment above

Reply:
Ok cool. I'll fix this by hand here, then sort out doing it properly as part of #59

Reply:
```suggestion
<p><h1>Input4MIPs CMIP6Plus datasets: v6.5.4</h1><p>
```